Packs/GitLab/ReleaseNotes/2_1_5.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### GitLab v2
+- Fixed an issue of documentation by removing general readme.
+- Updated the yml of **GitLabv2** by removing marketplace field.
```

Comment:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.9.40422*.
```

Reply:
added the last  note 'Updated the Docker image to: *demisto/python3:3.10.9.40422*.' to read me